ProductManagement.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace InventoryManagmentMongoDB
+{
+    public class ProductManagement
```

Comment:
this could be `ProductRepository `  in which u can interact with db context directly
or `ProductService `that calls methods from `ProductRepository`